src/ConfiguresApplication.php

```diff
@@ -146,13 +151,25 @@ protected function writeEnvironmentVariable($key, $value)
      */
     protected function reloadEnvironment()
     {
+        $adapters = [
+            new EnvConstAdapter(),
+            new PutenvAdapter(),
+            new ServerConstAdapter(),
+        ];
+
+        $repository = RepositoryBuilder::create()
+            ->withReaders($adapters)
+            ->withWriters($adapters)
+            ->make();
```

Comment:
Will this load into Laravel's environment, or is this our own repository? We want this to refresh Laravel's environment variables so that any other commands in the script read from the newly created `.env`.

If it's the latter, we might want to switch to this [`\Illuminate\Support\Env::getRepository`](https://github.com/laravel/framework/blob/cada3d21c7f127ba01bb949b2fa045b7faaedbca/src/Illuminate/Support/Env.php#L49-L70) method.

Reply:
What would be an ideal way to test this? I have a test command created, should I try doing a call to `config()` and grabbing an updated env variable after the call to `setEnvironmentVariable()`?

Reply:
Ok, within the `handle()` method for the test command I created in a new Laravel 7 project, after calling a number of methods from the Environmentalist trait, and also resetting the the default `DB_CONNECTION` env value, I added a call to dump `config('database.default')` at the end and got the new value. Seems like it's working as expected and using the updated env variables.

Reply:
Oh great! Glad it works!

It might be nice to pull Laravel's repository just to simplify this class a bit, but not necessary for merge.

Reply:
I can do that for the next update to support Laravel 8 :)